src/components/YearlyDataTab.tsx

```diff
@@ -43,6 +45,12 @@ export default class YearlyDataTab extends React.Component<YearlyDataTabProps, Y
             sortedByYear[year][catName].push(library);
           }
         }
+        // Order by month, within each category.
```

Comment:
I'd move this outside of `sortByYear` to not be so coupled. It might be useful later on to have a `sortByMonth` function.